pkg/cli/cli.go

```diff
@@ -145,7 +146,10 @@ func (c *CLI) buildCmd() error {
 	c.cmd = c.newRootCmd()
 
 	// Get project version and plugin keys.
-	if err := c.getInfo(); err != nil {
+	// Ignore 3-alpha error
+	switch err := c.getInfo(); {
```

Comment:
```suggestion
	switch err := c.getInfo(); err {
```

Reply:
Nope, that would only allow to compare exactly against `err`, it doesn't allow `errors.As`.